tutorials/spa-create-outlook-automation/spa-create-outlook-automation.md

```diff
@@ -0,0 +1,139 @@
+---
+author_name: Ramakrishnan Raghuraman
+author_profile: https://github.com/r3ksk
+title: Build your first automation using Outlook SDK of SAP Process Automation
+description: In this tutorial, you will be developing basic automation using Outlook SDK of SAP Process Automation. In the end, by using the activities in this Outlook SDK, you will know how to search outlook inbox or a specific email folder and download any attachments found in the email context to a specific local folder.
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform, tutorial>free-tier]
+primary_tag: software-product>sap-process-automation
+---
+
+## Details
+SAP Process Automation has native integration to several Microsoft Office products including Outlook, Excel SDK. In this Tutorial, you will explore how to use Outlook SDK to your automations.
+
+## Prerequisites
+
+- Complete the tutorial: [Subscribe to SAP Process Automation Using Booster in SAP BTP Free Tier](spa-subscribe-booster)
+- Complete the tutorial: [Install and Set Up the Desktop Agent](spa-setup-desktop-agent)
+- Familiarity to create an automation using SAP Process Automation
+
+### You will learn
+
+  - How to use Outlook SDK of SAP Process Automation to search the given email
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](About Outlook SDK in SAP Process Automation)]
+
+The Outlook SDK in SAP Process Automation, is a collection of activities allowing you to create automations using Microsoft Outlook.
```

Comment:
You use the name “Outlook SDK in SAP Process Automation” on this line, but in line 5, you say “Outlook SDK of SAP Process Automation.”  Please make this consistent.